src/main/java/lottogame/ui/LottoOutputView.java

```diff
@@ -34,14 +36,28 @@ private static String formatLottoNumbers(List<LottoNumber> numbers) {
     public static void printWinningStatistics(Map<Rank, Integer> winningResults) {
         System.out.println("당첨 통계");
         System.out.println("---------");
-        for (Rank rank : Rank.values()) {
-            if (rank != Rank.NONE) {
-                System.out.printf("%d개 일치 (%.0f원) - %d개%n",
-                        rank.getRank(),
-                        rank.getAmount(),
-                        winningResults.getOrDefault(rank, 0));
-            }
+        getWinningRanks().forEach(rank -> printRankStatistics(rank, winningResults));
+    }
+
+    private static List<Rank> getWinningRanks() {
+        return Arrays.stream(Rank.values())
+                .filter(rank -> !rank.isNone())
+                .sorted(Comparator.reverseOrder())
+                .collect(Collectors.toList());
+    }
+
+    private static void printRankStatistics(Rank rank, Map<Rank, Integer> winningResults) {
+        System.out.printf("%s (%.0f원) - %d개%n",
+                getMatchDescription(rank),
+                rank.getPrizeMoney(),
+                winningResults.getOrDefault(rank, 0));
+    }
+
+    private static String getMatchDescription(Rank rank) {
+        if (rank == Rank.SECOND) {
```

Comment:
이 부분도 객체에게 메시지를 보내서 처리해보면 어떨까요?
```suggestion
        if (rank.isSecond()) {
```